src/routes/MetaDetails/StreamsList/StreamsList.js

```diff
@@ -64,96 +64,106 @@ const StreamsList = ({ className, video, ...props }) => {
                 {
                     value: ALL_ADDONS_KEY,
                     label: t('ALL_ADDONS'),
-                    title: t('ALL_ADDONS')
+                    title: t('ALL_ADDONS'),
                 },
                 ...Object.keys(streamsByAddon).map((transportUrl) => ({
                     value: transportUrl,
                     label: streamsByAddon[transportUrl].addon.manifest.name,
                     title: streamsByAddon[transportUrl].addon.manifest.name,
-                }))
+                })),
             ],
             selected: [selectedAddon],
-            onSelect: onAddonSelected
+            onSelect: onAddonSelected,
         };
     }, [streamsByAddon, selectedAddon]);
     return (
         <div className={classnames(className, styles['streams-list-container'])}>
-            {
-                props.streams.length === 0 ?
-                    <div className={styles['message-container']}>
-                        <Image className={styles['image']} src={require('/images/empty.png')} alt={' '} />
-                        <div className={styles['label']}>No addons were requested for streams!</div>
+            {props.streams.length === 0 ? (
```

Comment:
It seems like your IDE did automatic indentation and formatting in lots of place, can you revert those?